### PR TITLE
Enhance README with examples

### PR DESCRIPTION
**Pull request by mattlindsey**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,8 +89,18 @@ data.main.temp # => 12
...
+client.current_city('Sydney')
+client.current_city('London, UK')
+client.current_city('London', 'UK')
+client.current_city('Albany')
+client.current_city('Albany, New York')
+client.current_city('Albany, New York', 'US')
+client.current_city('Albany, NY', 'US')
+client.current_city('Albany', 'New York', 'US')
+client.current_city('Albany', 'NY', 'US')
+client.current_city('Albany', 'NY')  # Bad: 2 letter state abbreviation without country will give Faraday::Resource not found
```
Comment (dblock) [inline README.md:101]: Say "2-letter state abbreviation w/o country will raise Faraday::ResourceNotFound" (this is the correct spelling of the error)

File: README.md [hunk 2]
```diff
@@ -89,8 +89,18 @@ data.main.temp # => 12
...
+
+client.current_weather(city: 'Albany', state: 'NY', country: 'US') # Good
```
Comment (dblock) [inline README.md:103]: Remove "Good"

File: README.md [hunk 3]
```diff
@@ -87,10 +87,21 @@ data.main.temp # => 12
 ```
 
 Returns weather by city, optional state (in the US) and optional ISO 3166 country code.
+Names that cannot be resolved will cause the API call to fail with Faraday::ResourceNotFound error.
```
Comment (dblock) [inline README.md:90]: Quote `Faraday::ResourceNotFound`

File: README.md [hunk 4]
```diff
@@ -87,10 +87,21 @@ data.main.temp # => 12
...
+client.current_city('Sydney')
+client.current_city('London, UK')
+client.current_city('London', 'UK')
+client.current_city('Albany')
+client.current_city('Albany, New York')
+client.current_city('Albany, New York', 'US')
+client.current_city('Albany, NY', 'US')
+client.current_city('Albany', 'New York', 'US')
+client.current_city('Albany', 'NY', 'US')
+client.current_city('Albany', 'NY')  # 2-letter state abbreviation w/o country will raise Faraday::ResourceNotFound"
```
Comment (dblock) [inline README.md:102]: There's an extra `"` at the end, remove.

**Review by dblock — CHANGES_REQUESTED** [review]
The extra " typo does need to be fixe ;) Thanks for hanging in here with me!

**Comment by mattlindsey** [comment]
> The extra " typo does need to be fixe ;) Thanks for hanging in here with me!
> 
> (no need for a changelog)

No problem.  Should be all set now.  :)

**Pull request merged**